README.md

```diff
@@ -14,8 +14,7 @@ A powerful and elegant C# library for Razer Chroma's SDK
 Getting started
 ---------------
 
-If you are a new developer and are looking for a helpful guide on how to get started, head on over to
-[the wiki page][getting-started] which describes getting Colore installed and running some example code.
+If you are a new developer and are looking for a helpful guide on how to get started, head on over to the [Documentation](https://chroma-sdk.github.io/Colore/docs/articles/getting-started.html) which describes getting Colore installed and running some example code.
```

Comment:
Nitpick: "Documentation" should be lowercase here as it's a regular word and lines should strive to be at most 120 characters in length.

Reply:
ok, but you approve the idea?

Reply:
Yeah I think it's a good idea to link to the docs version, at least for now as it is where effort should be put into documenting the library.